components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -249,9 +322,29 @@ def validate_config_key_existence(config, key):
     return value
 
 
-def validate_and_load_config_file(
+def validate_and_load_config_for_job_submission_script(
```

Comment:
can't think of a good name for this method. I don't want to call it `validate_and_load_config` because the usage of this method is only limited to the caller script under `package_utils/scripts`. I also feel it's ok to revert this change.

Reply:
Yeah, maybe it's better to revert cause I'm also noticing that the different instances of this code validate different directories.